src/components/dockerRegistry/Docker.tsx

```diff
@@ -73,7 +74,13 @@ export default function Docker({ ...props }) {
     useEffect(() => {
         _getInit()
     }, [])
-
+    if (!props.isSuperAdmin) {
+        return (
+            <div className="error-screen-wrapper flex column h-100">
+                <ErrorScreenNotAuthorized />
+            </div>
+        )
+    }
     if ((loading && !result) || clusterLoader) return <Progressing pageLoader />
     if (error) {
         showError(error)
```

Comment:
In case of errors, do you still want to show toast here?

Reply:
not showing